email_alert_service/listeners/listener.rb

```diff
@@ -34,8 +34,8 @@ def exit_on_exception(e, document_json, properties, delivery_info)
     # Rescue any exception, not just StandardError and subclasses.
     # We want to ensure that the process exits in such a situation, so we
     # explicitly call exit() after logging the error.
-    Airbrake.notify_or_ignore(e,
-      parameters: {
+    GovukError.notify(e.message,
```

Comment:
I think you want just `e`  here too to provide Sentry with all the data.

Reply:
Done